src/databricks/labs/ucx/source_code/base.py

```diff
@@ -178,12 +178,18 @@ class CurrentSessionState:
 
     @classmethod
     def from_json(cls, json: dict) -> CurrentSessionState:
+        mode_str = json.get('data_security_mode', None)
+        try:
+            data_security_mode = compute.DataSecurityMode(mode_str) if mode_str else None
+        except ValueError:
+            logger.error(f'Unknown data_security_mode {mode_str}')
```

Comment:
```suggestion
            logger.warning(f'Unknown data_security_mode {mode_str}')
```
error logs fail the workflow